src/Util/CapabilitiesUtil/CapabilitiesUtil.js

```diff
@@ -0,0 +1,61 @@
+import OlWMSCapabilities from 'ol/format/wmscapabilities';
+import OlSourceTileWms from 'ol/source/tilewms';
+import OlLayerTile from 'ol/layer/tile';
+
+import { get } from 'lodash';
+
+/**
+ * Helper Class to parse capabilities of WMS layers
+ *
+ * @class CapabilitiesUtil
+ */
+class CapabilitiesUtil {
+
+  /**
+   * @static parseWmsCapabilities - function
+   *
+   * @param {String} capabilitiesUrl Url to WMS capabilities document
+   *
+   * @return {Object} An object representing the WMS capabilities.
+   */
+  static parseWmsCapabilities(capabilitiesUrl) {
+    return fetch(capabilitiesUrl)
+      .then((response) => response.text())
+      .then((data) => {
+        const wmsCapabilitiesParser = new OlWMSCapabilities();
+        return wmsCapabilitiesParser.read(data);
+      });
```

Comment:
How are errors handled?

Reply:
These are handled by the method / statement handling Promise returned by `parseWmsCapabilities`. I've added an example in `AddWmsLayerEntry.example.jsx`

Reply:
I think adding a seperate error handler to every promise we use would bei even saver, but we don't do it anywhere. Lets open an issue therefore.